func_test.go

```diff
@@ -126,7 +134,7 @@ func TestRegisterLibFunc_Bool(t *testing.T) {
 	// this callback recreates the state where the return register
 	// contains other information but the least significant byte is false
 	cbFalse := purego.NewCallback(func() uintptr {
-		return 0x7F5948AE9A00
+		return uintptr(uint64(0x7F5948AE9A00) & uint64(^uintptr(0)))
```

Comment:
This number is too big for 32bit platforms so I masked it off to what fits in a uintptr. Should I add a comment explaining this?

Reply:
We could use smaller values but I wanted to keep the test unchanged on 64Bit platforms since they have larger registers and this test was found on a 64Bit platform